src/components/Card/Card.module.css

```diff
@@ -0,0 +1,12 @@
+.card {
+  border: solid black 1px;
+  border-radius: 25px;
+  padding: 10px;
+  display: grid;
+  grid-template-columns: 70px 160px 100px;
```

Comment:
style: In web, especially with responsiveness, I tend to avoid fixed units for box sizes, as much as possible. Why did you need this here? The size of the elements for each column + size `auto` + `align-content: space-evenly;` + some `gap` would be enough, no? :)